src/main/java/frc/robot/commands/AutoIntakeCommand.java

```diff
@@ -0,0 +1,33 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.filter.Debouncer;
+import edu.wpi.first.math.filter.Debouncer.DebounceType;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.CrocodileSubsystem;
+
+public class AutoIntakeCommand extends CommandBase{
+    private final CrocodileSubsystem crocodileSubsystem;
+    private final double speed;
+    private final Debouncer debouncer;
+    public AutoIntakeCommand(CrocodileSubsystem crocodileSubsystem, double speed){
```

Comment:
Do you want this to take a speed, or do you want to have a fixed intake speed for all autos? Will you end up with 6+ autos that all pass in 0.67, and have to change them all when we find it's better to intake at 0.52? If so, would you prefer to have that in one place?

I wonder if you want an "intake cube" vs. "intake cone" auto command, which you can tune properly for each game piece.

Reply:
all very valid points, i intended this to both be used in teleop and auto. in teleop, we may also have specified speeds for different game pieces, but the idea is to automate intaking.